src/Datadog.Trace/Datadog.Trace.csproj

```diff
@@ -23,6 +23,8 @@
 
   <ItemGroup>
     <PackageReference Include="LibLog" Version="5.0.6" PrivateAssets="All" />
+    <PackageReference Include="System.Buffers" Version="4.5.1" />
+    <PackageReference Include="System.Memory" Version="4.5.4" />
```

Comment:
Do these need to be PrivateAssets too? Or do they need to be vendored?

Reply:
I removed these completely. Older frameworks relay now on existing methods.